src/main/java/com/cburch/logisim/gui/start/Startup.java

```diff
@@ -522,10 +522,10 @@ private static RC handleArgPlain(Startup startup, Option opt) {
 
   private static RC handleArgGates(Startup startup, Option opt) {
     final var gateShape = opt.getValue();
```

Comment:
Please add `toLowerCase()` to `final var gateShape = opt.getValue()`:

```java
final var gateShape = opt.getValue().toLowerCase();
```

Reply:
done